dask/base.py

```diff
@@ -648,7 +650,22 @@ def tokenize(*args, **kwargs):
     """
     if kwargs:
         args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    start_time = default_timer()
+    result = md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    time_taken = default_timer() - start_time
+    warn_config_key = "tokenize.warn_time_secs"
+    warn_time_default = 2
+    warn_time = config.get(warn_config_key, default=warn_time_default)
+    if time_taken > float(warn_time):
+        warnings.warn(
+            "tokenize ran for {0:n} seconds."
+            " Configuration key {1} controls this threshold"
+            " and was set to {2} (default={3}).".format(
```

Comment:
Probably don't need to include the default in here. Which means we can inline the `default=` up above and avoid the temporary variable.

Can we also update the default to be in the config itself, rather than defining it here? I'm not sure how this is done, but I think it's doable.

Reply:
@TomAugspurger I took a look at the rest of the code and it seems that the other way defaults are set is at the top of the file below the imports with `config.update_defaults` as in `core.py`: `config.update_defaults({"array": {"chunk-size": "128MiB", "rechunk-threshold": 4}})`. Is that the best option here?